source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -243,6 +243,46 @@ number of writes may differ between source and destination.
 Transactions might not appear atomically on the destination cluster.
 Retryable writes may not be retryable on the destination cluster.
 
+.. _c2c-verifier-disclaimer:
+
+Embedded Verifier Disclaimer
```

Comment:
Right now this is a section under `Destination Clusters`. I don't think it belongs there. Can we move it to its own section and bring it to the top of this `mongosync behavior` page right before `Settings`?

Reply:
Fixed